docs/_documentations/remotedeploy-projects-eclipse.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: docs
+title: Eclipse
+description: Creating and importing projects
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing cloud connection, remote deployment of Codewind
+duration: 5 minutes
+permalink: remotedeploy-projects-eclipse
+type: document
+---
+
+# Creating and importing projects
+
+When you configure Codewind to be used remotely, you are ready to:
+
+1. Create a new project
+2. Import a project from your local Codewind instance to your remote Codewind instance on the cloud
+
+## Create a new project
+
+Go to the Codewind pane and click the Plus (+) icon on the new cloud deployment:
+
+![Creating new remote project](./images/remotedeploy-projects-eclipse/eclipse_create-new-project.png){:width="380"}
+
+Select the template type and enter a project name. Your new project is built and after a few seconds begins running in the cloud.
+
+## Import an existing local project to the cloud
+
+Copy an existing local project, for example `myFirstNodeProject`, over to the remote Codewind deployment:
+
+1\. Select the remote deployment and click `Add Existing Project`:
+
+![Add existing project](./images/remotedeploy-projects-eclipse/eclipse_add-project.png){:width="366"}
+
+2\. Go to the folder containing the local project files, click `Select a project from the workspace`, and select your project: 
+
+![Add to Codewind](./images/remotedeploy-projects-eclipse/eclipse_select-project.png){:width="654"}
+
+Codewind prompts you to confirm the project type, in this case, identified as `NodeJS`:
+
+![Confirm Project Type](./images/remotedeploy-projects-eclipse/eclipse_confirm-project-type.png){:width="654"}
+
+3\. Click `Finish`. Project files are copied over to the Codewind server, and the new `myFirstNodeProject` appears in the Codewind pane:
+
+![Project Added](./images/remotedeploy-projects-eclipse/eclipse_build-python-project.png){:width="196"}
+
+4\. Codewind begins building the code and the Docker image. Moments later the project image is uploaded to Docker Hub and used by your cloud deployment to provision a new pod:
+
+![Build Success](./images/remotedeploy-projects-eclipse/eclipse_build-python-project-successful.png){:width="189"}
+
+`myFirstNodeProject` on `CloudName1` cluster is now running and ready.
+
+## Move a project from one connection to another 
+
+With the Eclipse IDE, you can drag and drop projects from one connection to another. 
+
+Select the project in one connection and drag it into another connection: 
+
+![Drag NODEJS Project](./images/remotedeploy-projects-eclipse/eclipse_move-node-project-from-local-folder.png){:width="193"}
+
+When you remove the project from the first connection, you add it to the second connection:
```

Comment:
Change this to:
This action will remove the project from the first connection and add it to the second connection.